arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1433,15 +1413,15 @@
       }
     },
     "ErrorDetails": {
-      "description": "The properties related to the details of an error.",
+      "description": "Error details.",
```

Comment:
What purpose does specifying "required" properties serve in responses? I made sure all models which are parameters have the "required" properties specified wherever appropriate.